go/store/nbs/archive_build.go

```diff
@@ -366,27 +429,40 @@ type chunkGroup struct {
 	avgRawChunkSize            int
 }
 
-// chunkCmpScore wraps a chunk and its compression score for use by the chunkGroup.
+// chunkCmpScore wraps a chunk and its compression score for use by the chunkGroup. This score includes the
```

Comment:
Looks like Sami must have distracted you while you were finishing this sentence 😉🐕